ArcGISRuntimeSDKQt_CppSamples/LocalServer/LocalServerFeatureLayer/LocalServerFeatureLayer.cpp

```diff
@@ -60,37 +61,73 @@ void LocalServerFeatureLayer::componentComplete()
   // Set map to map view
   m_mapView->setMap(m_map);
 
-  QString dataPath = QDir::homePath() + "/ArcGIS/Runtime/Data";
+  // Check for ArcGIS Pro map package files
+  QString fileName = "PointsofInterest.mpkx";
+  QString dataPath = QDir::homePath() + "/ArcGIS/Runtime/Data/mpkx/" + fileName;
+
+  // Check to see if map package exists
+  if (!QFileInfo::exists(dataPath) || !QFileInfo(dataPath).isFile())
```

Comment:
[QFile::exists](https://doc.qt.io/qt-5/qfile.html#exists) might work here to condense this into a single check.